[core]/es_extended/server/commands.lua

```diff
@@ -132,6 +132,30 @@ end, false, {
 	}
 })
 
+ESX.RegisterCommand({ 'fix', 'repair' }, 'admin', function(xPlayer, args, showError)
+	local ped = GetPlayerPed(args.playerId.source)
+	local pedVehicle = GetVehiclePedIsIn(ped, false)
+	if not pedVehicle or GetPedInVehicleSeat(pedVehicle, -1) ~= ped then
+		showError(TranslateCap('not_in_vehicle'))
+		return
+	end
+	args.playerId.triggerEvent("esx:repairPedVehicle")
+	xPlayer.showNotification(TranslateCap('command_repair_success'), true, false, 140)
```

Comment:
Can you add notify to target. For example admin write command /fix 2. Yes you gave a notify, and target gave notify i think "Admin fix your car" or idk any message.

Reply:
and manage the fact that you can't give two notifications. If I have 2 id and I write /fix 2 I don't get two notifications I get one notification.

Reply:
Added !